node/cmd/guardiand/bech32.go

```diff
@@ -0,0 +1,285 @@
+package guardiand
+
+import (
+	"fmt"
+	"strings"
+)
+
+// TODO(cpatrick): replace this whole file with just "github.com/cosmos/cosmos-sdk/types"
```

Comment:
guardiand already imports `btcutil` https://github.com/wormhole-foundation/wormhole/blob/a6d76f43827530a538264af9daefffd7236df182/node/go.mod#L54

so we should not need to vendor this.

Reply:
Unfortunately cosmo-sdk is using a forked version of btcutil to decode bech32 so it's not straightforward to use the existing dependency.  I believe we need to eventually add cosmos-sdk as a dependency to integrate wormchain, so we can still eventually replace it with cosmos-sdk?